app/polymer/package.json

```diff
@@ -59,10 +58,12 @@
     "webpack-hot-middleware": "^2.21.2"
   },
   "devDependencies": {
-    "nodemon": "^1.17.1"
+    "nodemon": "^1.17.1",
+    "polymer-webpack-loader": "^2.0.1"
```

Comment:
Not sure that we need this dependency still, but perhaps I'm missing something here.

Reply:
Thanks! that doesn't sounds like a blocker tbh